### PR TITLE
Pros and cons

### PR DESCRIPTION
**Pull request by toniQ97**
# Description

Pros and cons component

#Screenshots

**Save on energy**
desktop 
image
mobile 
image

**Uswitch**
desktop  
image
mobile  
image

**Money**
dekstop  
image
mobile 
image

# Checklist

Pull request contains:

- [X] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [X] Includes theme changes for both Uswitch and money
- [X] Work has been tested in multiple browsers
- [X] PR description includes description of change
- [X] PR description includes screenshot of change
- [X] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [X] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/themes/bankrate/theme.json [hunk 1]
```diff
@@ -236,6 +240,16 @@
   },
 
   "compounds": {
+    "pros-and-cons": {
+      "title": {
+        "fontFamily": "helvetica",
+        "lineHeight": "md"
+      },
+      "item": {
+        "fontFamily": "helvetica",
```
Comment (domtronn) [inline src/themes/bankrate/theme.json:249]: We should just use the `body` font family from the theme, rather than introduce new typefaces

Comment (toniQ97) [reply src/themes/bankrate/theme.json:249]: According to design font should be Helvetica, but body in bankrate is 'Roboto, -apple-system, system-ui, system-ui, Arial, sans-serif', that's why I introduced new typeface

Comment (domtronn) [reply src/themes/bankrate/theme.json:249]: I imagine that's just a failing on the design system. We should keep body styles consistent - if they should be helvetica, we can update the `body` font family in the theme

@chrisunderdown thoughts/comments?

Comment (toniQ97) [reply src/themes/bankrate/theme.json:249]: Okay, I'll then update theme files to use body font

File: src/compounds/pros-and-cons/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/** @jsx jsx */
```
Comment (jonathan-fielding) [inline src/compounds/pros-and-cons/src/index.tsx:1]: why is this a seperate component and not just a variant of list?

**Comment by domtronn** [comment]
Closing due to #645

**Pull request closed without merging**